fenix/app/src/main/java/org/mozilla/fenix/compose/button/Button.kt

```diff
@@ -99,6 +109,8 @@ fun PrimaryButton(
     textColor: Color = FirefoxTheme.colors.textActionPrimary,
     backgroundColor: Color = FirefoxTheme.colors.actionPrimary,
     icon: Painter? = null,
+    iconModifier: Modifier = Modifier,
+    buttonEnabled: Boolean = true,
```

Comment:
```suggestion
    enabled: Boolean = true,
```
I think this should be understood since it's the same naming used for the Material `Button` composable.

Reply:
Done